os/hal/ports/WB32/LLD/GPIOv1/hal_pal_lld.c

```diff
@@ -96,6 +96,12 @@ void _pal_lld_setgroupmode(ioportid_t port, ioportmask_t mask, iomode_t mode) {
   uint32_t altr = (mode & PAL_WB32_ALTERNATE_MASK) >> 7;
   uint32_t current = (mode & PAL_WB32_CURRENT_MASK) >> 11;
 
+#ifdef SPI_SCK_PIN

```

Comment:
It should be noted that this is a QMK specific define, and would hint that the solution its trying to provide is incorrectly coupled.

Reply:
Okay, can you tell me what the right thing to do is？

Reply:
The problem has been dealt with qmk/qmk_firmware#17650.